paper.bib

```diff
@@ -19063,6 +19063,151 @@ @Article{Thrampoulidis2015LASSO
+  Booktitle                = {Biocomputing 2001},
+  Publisher                = {World Scientific},
+  Year                     = {2000},
+  Pages                    = {422--433}
+}
+
+@Article{Liu2013,
+  Title                    = {Change-Point Detection in Time-Series Data by Relative Density-Ratio Estimation},
+  Author                   = {Liu, S. and Yamada, M. and Collier, N. and Sugiyama, M.},
+  Journal                  = {Neural Networks},
+  Year                     = {2013},
+  Pages                    = {72--83},
+  Volume                   = {43}
+}
+
+@Article{Liu2014,
+  Title                    = {Direct Learning of Sparse Changes in {Markov} Networks by Density Ratio Estimation},
+  Author                   = {Liu, S. and Quinn, J. A. and Gutmann, M. U. and Suzuki, T. and Sugiyama, M.},
+  Journal                  = {Neural Computation},
+  Year                     = {2014},
+  Number                   = {6},
+  Pages                    = {1169--1197},
+  Volume                   = {26}
+}
+
+@Article{Liu2016a,
+  Title                    = {Support Consistency of Direct Sparse-Change Learning in {Markov} Networks},
+  Author                   = {Liu, S. and Suzuki, T. and Relator, R. and Sese, J. and Sugiyama, M. and Fukumizu, K.},
+  Journal                  = {Annals of Statistics},
+  Year                     = {2017},
+
+  Month                    = {06},
+  Number                   = {3},
+  Pages                    = {959--990},
+  Volume                   = {45}
+}
+
+@Book{Mackay2003,
+  Title                    = {Information theory, inference and learning algorithms},
+  Author                   = {MacKay, D.},
+  Publisher                = {Cambridge university press},
+  Year                     = {2003}
+}
+
+@Article{Shimodaira2000,
+  Title                    = {Improving predictive inference under covariate shift by weighting the log-likelihood function},
+  Author                   = {Shimodaira, A.},
+  Journal                  = {Journal of Statistical Planning and Inference},
+  Year                     = {2000},
+  Number                   = {2},
+  Pages                    = {227 - 244},
+  Volume                   = {90},
+
+  Doi                      = {https://doi.org/10.1016/S0378-3758(00)00115-4},
+  ISSN                     = {0378-3758},
+  Url                      = {http://www.sciencedirect.com/science/article/pii/S0378375800001154}
+}
+
+@Article{Sugiyama2008,
+  Title                    = {Direct Importance Estimation for Covariate Shift Adaptation},
+  Author                   = {Sugiyama, M. and Suzuki, T. and Nakajima, S. and Kashima, H. and von B\"unau, P. and Kawanabe, M.},
+  Journal                  = {Annals of the Institute of Statistical Mathematics},
+  Year                     = {2008},
+  Number                   = {4},
+  Pages                    = {699--746},
+  Volume                   = {60}
+}
+
+@Article{SUGIYAMA2011735,
```

Comment:
@Article{Sugiyama2011Least,
  author    = {Masashi Sugiyama and Taiji Suzuki and Yuta Itoh and Takafumi Kanamori and Manabu Kimura},
  title     = {Least-squares two-sample test},
  journal   = {Neural Networks},
  year      = {2011},
  volume    = {24},
  number    = {7},
  pages     = {735--751},
  month     = {sep},
  doi       = {10.1016/j.neunet.2011.04.003},
  publisher = {Elsevier {BV}},
}